assets/src/components/button/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Muriel-styled buttons.
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { Component } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+
+class NewspackButton extends Component {
+
+	/**
+	 * Render.
+	 */
+	render( props ) {
+		const { value } = this.props;
+		return (
+			<div className="newspack-button">
+				<Button { ...this.props } >
```

Comment:
The `@wordpress/components` `Button` component uses its children as the content, not `value`, and I think we should mirror this here. `children` is a prop, so this would mean changing the JSX here to simply:

```suggestion
				<Button className="newspack-button" { ...this.props } />
```